comictaggerlib/pagelisteditor.py

```diff
@@ -83,23 +116,26 @@ def __init__(self, parent):
 
         self.reset_page()
 
-        # Add the entries to the manga combobox
-        self.comboBox.addItem("", "")
-        self.comboBox.addItem(self.pageTypeNames[PageType.FrontCover], PageType.FrontCover)
-        self.comboBox.addItem(self.pageTypeNames[PageType.InnerCover], PageType.InnerCover)
-        self.comboBox.addItem(self.pageTypeNames[PageType.Advertisement], PageType.Advertisement)
-        self.comboBox.addItem(self.pageTypeNames[PageType.Roundup], PageType.Roundup)
-        self.comboBox.addItem(self.pageTypeNames[PageType.Story], PageType.Story)
-        self.comboBox.addItem(self.pageTypeNames[PageType.Editorial], PageType.Editorial)
-        self.comboBox.addItem(self.pageTypeNames[PageType.Letters], PageType.Letters)
-        self.comboBox.addItem(self.pageTypeNames[PageType.Preview], PageType.Preview)
-        self.comboBox.addItem(self.pageTypeNames[PageType.BackCover], PageType.BackCover)
-        self.comboBox.addItem(self.pageTypeNames[PageType.Other], PageType.Other)
-        self.comboBox.addItem(self.pageTypeNames[PageType.Deleted], PageType.Deleted)
+        # Add the entries to the page type combobox
+        self.cbPageType.addItem("", "")
+        self.cbPageType.addItem(self.pageTypeNames[PageType.FrontCover], PageType.FrontCover)
+        self.cbPageType.addItem(self.pageTypeNames[PageType.InnerCover], PageType.InnerCover)
+        self.cbPageType.addItem(self.pageTypeNames[PageType.Advertisement], PageType.Advertisement)
+        self.cbPageType.addItem(self.pageTypeNames[PageType.Roundup], PageType.Roundup)
+        self.cbPageType.addItem(self.pageTypeNames[PageType.Story], PageType.Story)
+        self.cbPageType.addItem(self.pageTypeNames[PageType.Editorial], PageType.Editorial)
+        self.cbPageType.addItem(self.pageTypeNames[PageType.Letters], PageType.Letters)
+        self.cbPageType.addItem(self.pageTypeNames[PageType.Preview], PageType.Preview)
+        self.cbPageType.addItem(self.pageTypeNames[PageType.BackCover], PageType.BackCover)
+        self.cbPageType.addItem(self.pageTypeNames[PageType.Other], PageType.Other)
+        self.cbPageType.addItem(self.pageTypeNames[PageType.Deleted], PageType.Deleted)
+
+        self.chkDoublePage.setText("D\U00000332ouble Page?")
+        self.lblBookmark.setText("Bookm\U00000332ark:")
```

Comment:
This should be changed in the ui file.
also Labels, menu items and tool bar items support using `&` for [mnemonics ](https://doc.qt.io/qt-5/qshortcut.html#details) which would be the preferred way to include the underline

Reply:
Is there a reason you do not use qt designer? it's included in `pip install qt5-applications~=5.15` if installing on linux it would be located at `~/.local/lib/python3.10/site-packages/qt5_applications/Qt/bin/designer`

Reply:
Also ampersands cannot be used directly in xml List of predefined entities in XML https://en.wikipedia.org/wiki/List_of_XML_and_HTML_character_entity_references#List_of_predefined_entities_in_XML

Reply:
The reason is that this is my first python program :-)
Moved to UI.